lib/app/utils/translation/l10n/intl_pl.arb

```diff
@@ -70,5 +70,7 @@
   "addWordFolderPage_addButtonText": "Dodaj",
   "addWordFolderPage_finishButtonText": "Zakończ",
   "addWordFolderPage_textFieldError": "Ups! Chyba coś tutaj zabrakło",
-  "folderContent_editWords": "Tryb edycji"
+  "folderContent_editWords": "Tryb edycji",
+  "your_native_language" : "Twój język ojczysty",
```

Comment:
We should follow a convention of naming the variables:
pageName_placement -> example: homePage_title